npm-frontend/spotify-server/app.js

```diff
@@ -98,17 +98,29 @@ app.get('/callback', function(req, res) {
           json: true
         };
 
+        var username, accountType;
         // use the access token to access the Spotify Web API
         request.get(options, function(error, response, body) {
           console.log(body);
-        });
+          username = body.uri;
+          accountType = body.product;
+
+          if(accountType == 'premium') {
```

Comment:
I have this implemented on the back end, we probably don't want it in both the front and back end. Which do you think we should keep?

Reply:
When I was doing the host id thing I realized the premium status was already right there so it's probably easiest to just keep the front end unless you feel strongly about using the back end.